P0009/P0009.bs

```diff
@@ -1001,13 +1003,9 @@ Table � — Layout mapping policy and layout mapping requirements
   <td></td>
 </tr>
 <tr>
-  <td>`m.get_extents()`</td>
-  <td>`E`</td>
-  <td>
-  
-  *Returns:* `e`.
-  
-  </td>
+  <td>`m.extents()`</td>
+  <td>convertable to `E`</td>
```

Comment:
Is there a good reason to not return an actual Extent? We'd rather have the more specific one. vector<bool> is a good example where this is a problem, since it returns something convertible to bool and not a bool. Which means that for example doing auto and than using the variable in a templated function is an issue.

Reply:
Convertable allows return by value or return by constant reference, which allows an implementation to avoid the overhead of being required to construct and return a copy.